editor/core/state/index.ts

```diff
@@ -57,6 +57,7 @@ const _state: State = {
 
   dancersArray: [],
   dancerPartIndexMap: {},
+  selectedLED: [],
```

Comment:
please use the existing state "selectedLEDs", it stores the **index** instead of the names